documentation/changes.md

```diff
@@ -0,0 +1,68 @@
+# Breaking Change Policy
+
+Customers and MVPs have told us many times in emphatic terms that breaking changes are a continuing problem in Azure PowerShell releases.  These changes cause an undue burden on users maintaining scripts to manage their Azure assets, and this causes reluctance in picking up the later versions of Azure PowerShell. It is clear that continuing, unmanaged breaking changes are unnacceptable to customers. 
+
+**To address customer concerns, Azure PowerShell is committing to limit breaking changes to one release per year and to treat any unplanned breaking change that ships as 
+a showstopper bug requiring a hotfix**.
+
+At the same time, we are introducing a new mechanism for [Pre-release AzureRM modules](#preview-modules).  This is detailed in the section [here](#preview-modules).
+
+This proposed policy is meant to apply to all stable Azure modules, starting with the 2.0 release. Details of the policy are as follows.
+- Each module in Azure PowerShell is versioned using [semantic versioning](http://semver.org).
+- Non-breaking changes and associated version updates are handled independently by module owners.  [Breaking changes](#breaking-change-definition) are scheduled for once a year and coordinated with the Azure SDK Team.
```

Comment:
@brendanburns.  This is codifying coordination that already happens.  For example, we coordinated changes in Tags and Confirmation to occur at the same time to avoid customers' scripts breaking twice.  The Confirmation change effectively happened a couple of months before, but we didn't remove deprecated parameters until ther next breaking change came along.  This kind of coordination would be desirable to customers, if it is possible.